### DISCUSSION TITLE
Waveform instead of seekbar and disable following tracks in the playlist

### DISCUSSION THREAD

**Discussion by ndw32-to** [Taiko2k/Tauon#1170] [Ideas]
Just a great player ideally lacks a waveform instead of a seekbar with a height of the entire bottom panel and an adaptively varying length depending on the size of the window. And also so that on each playlist you create, in the context menu or with a separate button, turn the queue on or off, so that you can manually turn on one track at a time. And checkboxes against each track in the playlist to disable playback. for example, I don't need to play five of the twenty tracks in the playlist, I'll just uncheck these five boxes and they won't play. Is it possible to do this? And what needs to be done to make it in the next release?

**Comment by Taiko2k** [comment]
More the issue is those are things I've consciously avoided for reasons, such as simplicity. I don't feel they fit with the design ethos of Tauon. Sometimes less is better and adding too many features makes things cluttered and overwhelming.

Though a waveform in particular, I feel quite strongly that they cause a psychoacoustic detraction.

**Reply by ndw32-to** [reply]
The question is not in psychoacoustics, but in visual performance and convenience of working in the waveform, you can see where the chorus is where the verse is, where is the dynamics of performance required at a certain point in time in the show, for example

**Reply by ndw32-to** [reply]
And I wouldn't say that Tauon is a simple player... A simple player is Amberol and one that has a waveform

**Comment by Solarunit** [comment]
I agree, I prefer a simple progress-bar vs Waveform

**Reply by ndw32-to** [reply]
as you will understand from the usual strip, where for example there is a sharp surge or a calm moment that is needed right now at this second for some show or performance

**Reply by Solarunit** [reply]
I think it's just not important

**Comment by ndw32-to** [comment]
Anyone who works at concert events will understand me

**Comment by ndw32-to** [comment]
![Снимок экрана от 2024-06-19 19-33-04](https://github.com/Taiko2k/Tauon/assets/172600035/77f6bbab-2a09-4560-a53c-cb48d98288a7)

And what about such a function?

**Reply by ndw32-to** [reply]
A checkbox with automatic jumping to the next track disabled

**Comment by ndw32-to** [comment]
![Снимок экрана от 2024-06-19 19-39-09](https://github.com/Taiko2k/Tauon/assets/172600035/b8847f90-a5f6-4fa2-9ea4-a496656e1c03)
And about the waveform, maybe you misunderstood me, I meant something like a waveform combined with a tape
